routers/web/repo/pull.go

```diff
@@ -1411,3 +1412,32 @@ func UpdatePullRequestTarget(ctx *context.Context) {
 		"base_branch": pr.BaseBranch,
 	})
 }
+
+// SetAllowEdits allow edits from maintainers to PRs
+func SetAllowEdits(ctx *context.Context) {
+	pr, err := models.GetPullRequestByIndex(ctx.Repo.Repository.ID, ctx.ParamsInt64(":index"))
+	if err != nil {
+		if models.IsErrPullRequestNotExist(err) {
+			ctx.NotFound("GetPullRequestByIndex", err)
+		} else {
+			ctx.ServerError("GetPullRequestByIndex", err)
+		}
+		return
+	}
+
+	if !ctx.IsSigned || !pr.Issue.IsPoster(ctx.Doer.ID) {
```

Comment:
question: should we instead of poster check if doer has write access to repo?

hack idea:
 - create a pull from a repo you dont own to a fork you do
 - grant access to form maintainer
 - push commits to a repo you had no permissions before :/

Reply:
oh no - hack works